theano/tensor/basic.py

```diff
@@ -3139,7 +3141,9 @@ def var(input, axis=None, keepdims=False):
     centered_input = input - mean_input
 
     # return the mean sqr
-    v = mean((centered_input ** 2), axis, keepdims=keepdims)
+    #divisor = theano.tensor.cast(prod(input.shape[axis]) - ddof, theano.config.floatX)
+    divisor = prod(input.shape[axis]) - ddof
```

Comment:
if ddof is 0, don't do the add. Do something like:
divisor = prod()
if ddof:
    divisor = divisor - ddof